homeassistant/components/automation/__init__.py

```diff
@@ -522,7 +622,9 @@ async def _async_process_config(
         ]
 
         for list_no, config_block in enumerate(conf):
+            raw_config = None
             if isinstance(config_block, blueprint.BlueprintInputs):  # type: ignore
+                # TODO: handle raw config for blueprints
```

Comment:
Raw config is post-substitute and before it's validated at line 635

```python
raw_config = blueprint_inputs.async_substitute()
```

Reply:
Thanks, fixed!